build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidInternalDir>$(OutputPath)\lib\mandroid-internal</_MandroidInternalDir>
```

Comment:
I forget these things, unless written down. :-)

Using `remap-assembly` is fine. The point is, however, that we want a commit that is a "cohesive whole," and thus it doesn't make sense to add a "broken" `mdoc.exe` which can't run, because it's referencing an assembly which doesn't exist (unless on mono -- Linux or macOS).

Additionally, `mandroid-internal` is a bit of a "monodroid-ism" which may not make sense externally. Perhaps we should use `mandroid-to-remap`?